recipes/behaviortree.cpp/all/patches/4.0.2-0002-use-cci.patch

```diff
@@ -0,0 +1,47 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index b713372..3609e2c 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -59,9 +59,9 @@ endif()
+ 
+ #---- Find other packages ----
+ find_package(Threads REQUIRED)
+-find_package(ZMQ)
++find_package(ZeroMQ)
```

Comment:
This can be changed in a recipe with CMakeDeps using the `set_property` in the generate method